src/components/Search/Search.js

```diff
@@ -1,29 +1,100 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import SearchIcon from '@material-ui/icons/Search';
 import InputBase from '@material-ui/core/InputBase';
-
 import { FormattedMessage } from 'react-intl';
+import { Link, useHistory } from 'react-router-dom';
 import useStyles from './searchStyles';
+import fuse from './FuseSearch';
 
 function Search() {
   const classes = useStyles();
+  const history = useHistory();
+  const [searchQuery, setSearchQuery] = useState('');
+  const [showResults, setShowResults] = useState(false);
+  const [searchResults, setSearchResults] = useState([]);
+  const handleChange = event => {
+    setSearchQuery(event.target.value);
+  };
+
+  useEffect(() => {
+    const results = fuse.search(searchQuery).slice(0, 15);
```

Comment:
Can be saved to a constant as **MAX_SEARCH_RESULTS**
and can also be done in a separate function maybe just to understand what it is.
```suggestion
    const results = fuse.search(searchQuery).slice(0, MAX_SEARCH_RESULTS);
```